doc/distrib/License.rtf

```diff
@@ -1,526 +1,119 @@
+\cf1\b\f0\par
+OpenSans font from Google\par
+{\cf0\b0\f2{\field{\*\fldinst{HYPERLINK http://www.google.com/fonts/specimen/Open+Sans }}{\fldrslt{http://www.google.com/fonts/specimen/Open+Sans\ul0\cf0}}}}\cf2\b0\f5\fs22\line {\cf0\f2{\field{\*\fldinst{HYPERLINK http://www.apache.org/licenses/LICENSE-2.0.html }}{\fldrslt{http://www.apache.org/licenses/LICENSE-2.0.html\ul0\cf0}}}}\cf0\f3\fs22\par
+\cf2\par
+\cf1\kerning1\b\f0 Prism\par
+{\cf0\b0\f2{\field{\*\fldinst{HYPERLINK http://msdn.microsoft.com/en-us/library/gg406140.aspx }}{\fldrslt{http://msdn.microsoft.com/en-us/library/gg406140.aspx\ul0\cf0}}}}\cf2\b0\f5\fs22\line {\cf0\f2{\field{\*\fldinst{HYPERLINK http://msdn.microsoft.com/en-us/library/gg405489(PandP.40).aspx }}{\fldrslt{http://msdn.microsoft.com/en-us/library/gg405489(PandP.40).aspx\ul0\cf0}}}}\cf0\f2\fs22\par
+\cf2\kerning0\f3\par
+\cf1\b\f0 Python Standard Library\par
+{\cf0\b0\f2{\field{\*\fldinst{HYPERLINK https://docs.python.org/2.7/library/ }}{\fldrslt{https://docs.python.org/2.7/library/\ul0\cf0}}}}\cf0\b0\f2\fs22\par
+{{\field{\*\fldinst{HYPERLINK https://docs.python.org/2.7/license.html }}{\fldrslt{https://docs.python.org/2.7/license.html\ul0\cf0}}}}\cf2\kerning1\f2\fs22\par
+\f3\par
+\cf1\kerning0\b\f0 Python.NET\par
+{\cf0\b0\f2{\field{\*\fldinst{HYPERLINK https://github.com/pythonnet/pythonnet/blob/master/LICENSE }}{\fldrslt{https://github.com/pythonnet/pythonnet/blob/master/LICENSE\ul0\cf0}}}}\cf2\kerning1\b0\f3\fs22\par
+\cf0\par
+\cf1\b\f0 RestSharp\par
+{\cf0\b0\f2{\field{\*\fldinst{HYPERLINK http://www.apache.org/licenses/LICENSE-2.0 }}{\fldrslt{http://www.apache.org/licenses/LICENSE-2.0\ul0\cf0}}}}\cf0\b0\f2\fs22\par
+\f3\par
+\cf1\b\f0 SharpDX\par
+{\cf0\b0\f2{\field{\*\fldinst{HYPERLINK https://github.com/sharpdx/SharpDX/blob/master/LICENSE }}{\fldrslt{https://github.com/sharpdx/SharpDX/blob/master/LICENSE\ul0\cf0}}}}\cf0\b0\f3\fs22\par
+\par
+\cf5\b\f0 SimplexNoise\par
+{\cf0\b0\f2{\field{\*\fldinst{HYPERLINK https://unlicense.org/ }}{\fldrslt{https://unlicense.org/\ul0\cf0}}}}\cf0\b0\f2\fs22\par
+\par
+\cf5\b\f0 StarMath\par
+{\cf0\b0\f2{\field{\*\fldinst{HYPERLINK https://github.com/DesignEngrLab/StarMath/blob/master/LICENSE }}{\fldrslt{https://github.com/DesignEngrLab/StarMath/blob/master/LICENSE\ul0\cf0}}}}\cf0\b0\f2\fs22\par
+\f3\par
+\cf1\b\f0 Xceed Extended WPF Toolkit\f1\par
+{{\field{\*\fldinst{HYPERLINK "https://opensource.org/licenses/ms-pl.html"}}{\fldrslt{\ul\cf4\cf0\b0\f2 Microsoft Public License}}}}\cf0\b0\f3\fs22\par
+{\f2{\field{\*\fldinst{HYPERLINK https://github.com/xceedsoftware/wpftoolkit/blob/master/license.md }}{\fldrslt{https://github.com/xceedsoftware/wpftoolkit/blob/master/license.md\ul0\cf0}}}}\f3\fs22\par
+
+\pard\nowidctlpar\sb240\par
+}

```

Comment:
For some reason seems there are way more content in the previous version? Is this IDE related? @mjkkirschner  should know more here

Reply:
yeah, the same RTF view can be created with different elements of course - we should really test this in the application though as .net renders rtf kinda funny.

Reply:
I noticed that too. I initially edited the file in WordPad and opened it in Dynamo -> About. Things looked good except for a few links I added that were invisible! To fix that I had to play with the "RTF tags" until they decided to appear.